docs/output.md

```diff
@@ -62,14 +62,20 @@ Most plots are included in the HTML report (see above), but are also included in
     - `[array platform].annotation.tsv`: Annotations derived from an array platform
     - `[GTF name].anno.tsv`: Species wise annotations derived from a GTF in RNA-seq analysis
   - `processed_abundance/`: Directory containing processed abundance values from initial processing from e.g. DESeq2 or Affy:
-    - `[contrast_name].normalised_counts.tsv`: Normalised counts table (DESeq2)
-    - `[contrast_name].vst.tsv`: Normalised counts table with a variance-stabilising transform (DESeq2)
+    - `[contrast_name].normalised_counts.tsv`: Normalized counts table (DESeq2). Rows are ensembl IDs and columns are samples.
+    - `[contrast_name].vst.tsv`: Normalized counts table with a variance-stabilizing transform (DESeq2)
     - `raw.matrix.tsv`: RMA background corrected matrix (Affy)
-    - `normalised.matrix.tsv`: RMA background corrected and normalised intensities matrix (Affy)
+    - `normalised.matrix.tsv`: RMA background corrected and normalized intensities matrix (Affy)
   - `differential/`: Directory containing tables of differential statistics reported by differential modules such as DESeq2
+
+    - `[contrast_name].deseq2.results.tsv`: Results of DESeq2 differential analysis (RNA-seq)
+    - `OR [contrast_name].limma.results.tsv`: Results of Limma differential analysis (Affymetrix arrays) including ensembl ID, log2FC, and adjusted P. Value
```

Comment:
There's a lot of conditional logic in the workflow that causes certain files to appear in certain circumstances. We don't need to express that conditionality here.